### PR TITLE
Set the cursor to 'crosshair' when expected to draw

### LINKED ISSUE #102: Change cursor to something else while expected to draw
Right now it’s unclear what you are supposed to do

### PR DESCRIPTION
**Pull request by azz**
And reset it back to default on mouse up.

Fixes #102

### CODE REVIEW

File: src/index.tsx [hunk 1]
```diff
@@ -1181,7 +1182,6 @@ class App extends React.Component<{}, AppState> {
             let resizeHandle: string | false = false;
             let isDraggingElements = false;
             let isResizingElements = false;
-            const cursorStyle = document.documentElement.style.cursor;
```
Comment (azz) [inline src/index.tsx]: I'm not sure what state this was preserving? It seemed to always be `''` on my machine.

Comment (vjeux) [reply src/index.tsx]: I didn't write this but I personally have the habit of restoring things to whatever they were before.

**Comment by vjeux** [comment]
Could you also reset it when we are inserting a text element by any chance. Because we are doing a prompt, onMouseUp doesn't get triggered. (Well, now with #144 it does after the next mousedown but that's still a bit too late).

**Comment by vjeux** [comment]
image

**Comment by azz** [comment]
Done, also set the text entry to use the `'text'` cursor (I-beam) instead.

**Pull request merged**